api/environments/identities/views.py

```diff
@@ -55,10 +129,23 @@ def get_environment_from_request(self):
 
     def perform_create(self, serializer):
         environment = self.get_environment_from_request()
+        if environment.project.enable_dynamo_db:
```

Comment:
Feels to me like this logic should just live in the serializer. It also does feel like this (along with my comment above regarding pagination) means we should be looking at a separate endpoint for retrieving identities from dynamo. Feels like we're having to write a lot of conditional logic that's getting quite messy.